src/development/troubleshoot.md

```diff
@@ -13,9 +13,9 @@ platform redeploy
 Do not trigger a redeploy if there are builds in a "Pending" state, as these will block deployment. Wait for those builds to complete.
 
 
-## HTTP 502 response
+## HTTP responses "502 bad gateway" or "503 service unavailable"
 
-If you receive HTTP 502 responses from your site, it means that your site is running out of workers or the application process is crashing.  Here are the typical causes:
+If you receive HTTP responses from your site, it means that your site is running out of workers or the application process is crashing.  Here are the typical causes:
```

Comment:
Receive "these" responses".

Also, it doesn't guarantee those are the reasons; it just suggests they could be.